pupa/tests/importers/test_bill_importer.py

```diff
@@ -50,7 +51,15 @@ def test_full_bill():
     # import bill
     oi = OrganizationImporter('jid')
     oi.import_data([org.as_dict(), com.as_dict()])
-    BillImporter('jid', oi).import_data([oldbill.as_dict(), bill.as_dict()])
+
+    pi = PersonImporter('jid')
+    pi.json_to_db_id['person-id'] = 'person-id'
```

Comment:
just a suggestion, maybe comment why this is needed in the code.. I can see this seeming wrong to someone later